src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Main{
```

Comment:
The class name 'Main' should not have an empty line at the beginning of its definition.